src/chi/components/time-picker/time-picker.scss

```diff
@@ -37,7 +37,7 @@
       .chi-time-picker__period {
         border-radius: $border-radius-base;
         cursor: pointer;
-        margin: 0.25rem;
+        margin: 0 0.25rem;
```

Comment:
Removing margin-top on time picker items will cause the items to sit flush with the content container edge.
![image](https://user-images.githubusercontent.com/130654/103566740-6eaf7780-4e77-11eb-88f5-2a203a1e204a.png)

To achieve the same visual effect without margin, you will need to apply `padding-top: 0.25rem;` to `chi-time-picker__periods`, `chi-time-picker__minutes`, and `chi-time-picker__hours`

Reply:
Done 👍